setup-container.sh

```diff
@@ -3,8 +3,6 @@
 DOCKER_SONIC_MGMT="docker-sonic-mgmt"
 DOCKER_REGISTRY="sonicdev-microsoft.azurecr.io:443/"
 VAR_FILE="ansible/vars/docker_registry.yml"
-USER_KEY="docker_registry_username"
```

Comment:
No need to change this file, I have already merged #2068 which accomplishes anonymous pull in this script.

Reply:
Rebased and reverted.